src/v/cloud_storage/tests/remote_partition_test.cc

```diff
@@ -869,27 +886,39 @@ scan_remote_partition_incrementally(
 
     storage::log_reader_config reader_config(
       base, max, ss::default_priority_class());
+
+    std::seed_seq seq{11};
+    std::mt19937 re(seq);
+    std::uniform_int_distribution<unsigned> dist;
```

Comment:
why not use `random_generators::...`? I think in that case tests print out the seed so its reproducible if needed.

Reply:
fixed